package.json

```diff
@@ -38,11 +38,12 @@
     "@ethersproject/strings": "^5.6.1",
     "@zk-kit/identity": "^1.4.1",
     "@zk-kit/incremental-merkle-tree": "^0.4.3",
-    "circomlibjs": "0.1.6",
+    "circomlibjs": "^0.0.8",
```

Comment:
`import { poseidon } from "circomlibjs"`
In the 0.1.6 version of circomjs, poseidon isn't imported properly. In this version, poseidon becomes undefined.
Maybe the naming was changed. Temporarily, I changed it to the previous version.

Reply:
circomlibjs 0.1.7 with ffjavascript 0.2.55 should work hmm. I believe you have to build.

Reply:
https://github.com/iden3/circomlibjs/blob/4f094c5be05c1f0210924a3ab204d8fd8da69f49/test/poseidon.js